File: ConduitC#App/conduit-sample-app/api/requests/LoginRequest.cs
using System.Text;
using System.Net.Http;

using Newtonsoft.Json;

using user = entity.User;

namespace request
{
    public class LoginRequest : IRequest
    {
        private user user;
        private StringContent payload;
        public LoginRequest(user userValue)
        {
            user = userValue;
        }
        public StringContent createPayload()
        {
            payload = new StringContent(JsonConvert.SerializeObject(user), Encoding.UTF8, "application/json");
            return payload;
        }
        public StringContent getPayload() { return payload; }

        public override string ToString()
        {
            return "LoginRequest {" +
             "user='" + user.ToString() + "}";
        }

    }
}

File: ConduitJavaApp/src/main/java/com/conduit/sample/api/requests/LoginRequest.java
package com.conduit.sample.api.requests;

import org.json.JSONObject;

public class LoginRequest implements IRequest {

    private String email;
    private String password;
    private JSONObject jsonObject = new JSONObject();

    @Override
    public JSONObject getPayload() {
        return jsonObject;
    }


    public LoginRequest(String email, String password) {
        this.email = email;
        this.password = password;
    }

    public JSONObject createPayload() {
        jsonObject.put("email", email);
        jsonObject.put("password", password);

        return jsonObject;
    }
}
